corehq/apps/es/queries.py

```diff
@@ -93,11 +93,15 @@ def ids_query(doc_ids):
     return {"ids": {"values": doc_ids}}
 
 
-def match(search_string, field, fuzziness="AUTO"):
+def match(search_string, field, fuzziness="AUTO", operator=None):
```

Comment:
Tests for this new functionality should live in `corehq/apps/es/tests/test_queries.py`.

Reply:
Same as my comment above, are you suggesting I add new, more specific tests or that I move existing ones?

Reply:
Just moving the existing one is fine. The diff view had me thinking you added more than one test to `corehq/apps/case_search/tests/test_case_search_es_queries.py`, but I see that's not the case. To simplify my ask: everything you put in that test (in commit c06041762f729be5cdcd3c34b00cf76cd15dc7a5) should be in `corehq/apps/es/tests/test_queries.py` instead.

Reply:
Gotcha, thanks for clarifying Joel. Test moved in my most recent commit!